sympy/polys/ring_series.py

```diff
@@ -572,6 +574,111 @@ def _coefficient_t(p, t):
+
+    The series should be of form $p(x) = a_1 x + a_2 x^2 + ...$ where $a_1$ is invertible.
+
+    Parameters
+    ==========
+
+    p : :class:`~.PolyElement` to be inverted.
+    x : :class:`~.PolyElement` with respect to which ``p`` is inverted.
+    n : number of expansion terms to be computed.
+
+    Note
+    ====
+
+    The algorithm adapts the Newton method to solve a general equation
+    $$
+    f(x)=y.
+    $$
+    By Taylor's expansion:
+    $$
+    f(x+h) = f(x) + f'(x) h + O(h^2).
+    $$
+    If $f(x) = y + O(y^{k+1})$ then with $h = -\frac{f(x)-y}{f'(x)} = O(y^{k+1})$ we obtain:
+    $$
+    f(x+h)-y = O(h^2) = O(y^{2k+2}).
+    $$
+    Thus, updating $x\gets x-h$ increases the accuracy from $k$ to $2k+1$ terms.
+
+    Examples
+    ========
+
+    The example below shows how to invert the expresion $f=\mathrm{e}^{x}-1$ around $x=0$.
+    First we truncate the series of $f$, then convert it to fit the polynomial ring - in this case $Q[x]$, and finally call the method:
+
+    >>> from sympy import exp
+    >>> from sympy.abc import x
+    >>> from sympy.polys.ring_series import rs_fast_series_reversion
+    >>> f = exp(x)-1
+    >>> n = 4
+    >>> f = f.series(x,0,n).removeO()
+    >>> Rx = f.as_poly(x,field=True).domain[x]
+    >>> fr = Rx.from_sympy(f)
+    >>> xr = Rx.from_sympy(x)
+    >>> rs_fast_series_reversion(fr,xr,n)
+    1/3*x**3 - 1/2*x**2 + x
+
+    The example below show few first explicit coefficients of the inversion expansion, namely:
+
+    >>> import sympy as sm
+    >>> from sympy.polys.ring_series import rs_fast_series_reversion
+    >>> a = sm.IndexedBase('a')
+    >>> x = sm.symbols('x')
+    >>> f = sum(a[k]*x**k for k in range(1,4))
+    >>> R = f.as_poly(x,field=True).domain
+    >>> Rx = R[x]
+    >>> fr = Rx.from_sympy(f)
+    >>> xr = Rx.from_sympy(x)
+    >>> rs_fast_series_reversion(fr,xr,n).as_expr()
+    x**3*(-a[3]/a[1]**4 + 2*a[2]**2/a[1]**5) - x**2*a[2]/a[1]**3 + x/a[1]
+
+    References
+    ==========
+
+    .. [1] Brent, Richard P., and Hsiang T. Kung. "Fast algorithms for manipulating formal power series." Journal of the ACM (JACM) 25, no. 4 (1978): 581-595.
+    .. [2] Weisstein, Eric W. "Series Reversion." From MathWorld--A Wolfram Web Resource. https://mathworld.wolfram.com/SeriesReversion.html
+    """
+
+    f = p
+    Rx = f.ring
+    df = f.diff(x)
+    df_inv = rs_series_inversion(df,x,n)
+
+    # inject the auxiliary variable y
+    y = Dummy('y')
+    Rx = Rx.to_domain()
+    Rxy = Rx.inject(y)
+    df_inv = Rxy.convert_from(df_inv,Rx)
+    xr = Rxy.convert_from(x,Rx)
+    f = Rxy.convert_from(f,Rx)
+    yr = Rxy.from_sympy(y)
+
+    # check if inversion exists
+    assert Rx.domain.is_zero(f.coeff(xr**0)) # no constant term
+    assert Rx.domain.is_unit(f.coeff(xr)) # the first coeff is unit
+
+    # do Newton updates
+    fn_update = xr-(f-yr)* df_inv
+    x_series = Rxy.zero
+    k = 0
+    while k < n:
+        x_series = rs_subs(fn_update, {xr:x_series},yr,min(2*k+2,n))
```

Comment:
SymPy code loosely follows PEP 8 style which means having a space after a comma and usually around operators like `+` and `-` e.g.:
```python
x_series = rs_subs(fn_update, {xr:x_series}, yr, min(2*k + 2, n))
```
This style is not completely strict and some parts of the codebase use different styles but if you look around this file you can see that it does have this kind of spacing everywhere else.